lib/biran/config_defaults.rb

```diff
@@ -20,7 +20,7 @@ def app_defaults_init
     end
 
     def app_base
-      @app_base ||= ENV['APP_ROOT'] || app_config_defaults[:app][:root_path]
+      @app_base ||= ENV['BIRAN_APP_BASE_PATH'] || app_config_defaults[:app][:base_path] || app_config_defaults[:app][:root_path]
```

Comment:
This line allows for the old config option to still exist in `config/app_config.yml`. Won't break any of our existing configs if we forget to change that line.

Reply:
but `app_config_defaults[:app][:root_path]` will always be `nil` right?

Reply:
Going forward yes. But right now, for example, if we use this gem in our existing apps, we set `[:app][:root_path]`. So if we don't change that config to be `[:app][:base_path]` the gem will still work for us. Its like a catch for legacy config. That last condition can be removed later. I tested this with
```
defaults: &defaults
  app: &app_defaults
    root_path: <%= Rails.root %>
```
and
```
defaults: &defaults
  app: &app_defaults
    base_path: <%= Rails.root %>
```
To verify I get same results when generating files. I believe `base_path` should be preferred in this case because we end up taking that value and either appending things to it or using as is and converting to something else internally.